integration_tests/tests/s3-invoice-standardised-line-item-tests.ts

```diff
@@ -258,8 +258,8 @@ export const listS3Keys = async (
   prefix: string
 ): Promise<string[]> => {
   const result = await listS3Objects({ bucketName, prefix });
-  if (result.Contents === undefined) return [];
-  const keys: Array<string | undefined> = result.Contents.map(({ Key }) => Key);
+  if (result === undefined) return [];
+  const keys: Array<string | undefined> = result.map(({ key }) => key);
```

Comment:
Looking at `listS3ObjectsBasic`, where `result` ultimately comes from, the return value is initialised as an empty array then each result is pushed into it so there shouldn't be a case where `result` is undefined. 
You can also get away without the explicit typing of `keys`, `const keys = result.map(({ key }) => key);` is fine :)

Reply:
done, thanks